src/EndmemberLibrary.jl

```diff
@@ -66,18 +66,22 @@ end
 
 function get_good_bands_mask(wavelengths::Array{Float64}, wavelength_pairs)
     good_bands = ones(Bool, length(wavelengths))
-
-    for wvp in wavelength_pairs
-        wavelength_diff = wavelengths .- wvp[1]
-        wavelength_diff[wavelength_diff .< 0] .= maximum(filter(!isnan, wavelength_diff))
-        lower_index = nanargmin(wavelength_diff)
-
-        wavelength_diff = wvp[2] .- wavelengths
-        wavelength_diff[wavelength_diff .< 0] .= maximum(filter(!isnan, wavelength_diff))
-        upper_index = nanargmin(wavelength_diff)
-        good_bands[lower_index:upper_index] .= false
+    
+    if length(wavelengths) > 7 
+        for wvp in wavelength_pairs
+            wavelength_diff = wavelengths .- wvp[1]
+            wavelength_diff[wavelength_diff .< 0] .= maximum(filter(!isnan, wavelength_diff))
+            lower_index = nanargmin(wavelength_diff)
+
+            wavelength_diff = wvp[2] .- wavelengths
+            wavelength_diff[wavelength_diff .< 0] .= maximum(filter(!isnan, wavelength_diff))
+            upper_index = nanargmin(wavelength_diff)
+            good_bands[lower_index:upper_index] .= false
+        end
+    else
+        good_bands = good_bands
+    
```

Comment:
Can you comment on what this is looking to address?

Reply:
This allows us to use MODIS/Landsat bands in the unmixing procedures, if a user provides a small number of bands. Not critical for unmixing of EMIT/AVIRIS imagery.

Reply:
Got it - I'm going to propose that we change this to a modification of the wavelength_pairs in unmix.jl, and turn it into an argument, to avoid hardcoding.